xcell/cls/theory.py

```diff
@@ -203,4 +249,4 @@ def get_ccl_cl_covNG(self, ccl_trA1, ccl_trA2, ellA,
                                       tkka=tkk, fsky=fsky,
                                       cltracer3=ccl_trB1['ccl_tr'],
                                       cltracer4=ccl_trB2['ccl_tr'],
-                                      ell2=ellB)
+                                      ell2=ellB, integration_method='spline')
```

Comment:
Should we make `integration_method` an input option?

Reply:
There are lots of other default options in these CCL functions we're not letting users modify. This is just because it ensures that the calculation of the covNG won't fail because of a gsl integration error. In 99.9% of the cases that doesn't happen, but because we need to compute so many covariance matrices, I thought it best to use spline integration to avoid the odd case when it happens.

Reply:
Ok. Makes sense. Thanks!